simplq/src/api/auth.js

```diff
@@ -35,7 +35,7 @@ const getAuthHeaderValue = async (auth) => {
  * @returns — makeAuthedRequest async request
  */
 const useMakeAuthedRequest = () => {
-  const auth = useAuth0();
+  const auth = useAuth0 ();
 
```

Comment:
This is a tiny thing, but if you're going to make more changes, removing these spaces between the function name and the parentheses would be nice.

Reply:
This file and a few other only have styling changes, if that's not intentional, it would be a good idea to revert them. Let's try to only focus on logic changes here, since it's already a big PR.

Reply:
I have resolved all those in my latest commit. Thanks for the guidance